pyproject.toml

```diff
@@ -9,3 +9,24 @@
     #   remember to remove the extra newlines in the rendered changelog.
     title_format = ""
 
+    [[tool.towncrier.type]]
+        directory = "breakingchange"
+        name = "Breaking Changes"
+        showcontent = true
+
+    [[tool.towncrier.type]]
+        directory = "feature"
+        name = "New & Improved"
+        showcontent = true
+
+    [[tool.towncrier.type]]
+        directory = "bugfix"
+        name = "Fixed"
+        showcontent = true
+
+    [[tool.towncrier.type]]
+        directory = "misc"
+        name = "Misc"
+        showcontent = true
+
+
```

Comment:
I'm unfortunately no great expert on the `toml` format, but do we need that many trailing lines here at the end?